server/channels/web/oauth_test.go

```diff
@@ -404,7 +404,7 @@ func TestMobileLoginWithOAuth(t *testing.T) {
 	var siteURL = "http://localhost:8065"
 	th.App.UpdateConfig(func(cfg *model.Config) {
 		*cfg.ServiceSettings.SiteURL = siteURL
-		cfg.NativeAppSettings.AppCustomURLSchemes = append(cfg.NativeAppSettings.AppCustomURLSchemes, "mmauth://")
+		*cfg.GitLabSettings.Enable = true
```

Comment:
The addition of `GitLabSettings.Enable` seems unexpected relative to https://github.com/mattermost/mattermost/pull/24554 -- is it intentional?

Reply:
Yes without it, the method returns an error that no sso service is enabled